charts/galoy/templates/admin-panel-ingress.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if (index .Values "admin-panel").ingress.enabled }}
+apiVersion: {{ include "common.capabilities.ingress.apiVersion" . }}
+kind: Ingress
+metadata:
+  name: {{ include "admin-panel.fullname" . }}
+  labels:
+    app: {{ template "admin-panel.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+  annotations:
+    {{- if (index .Values "admin-panel").ingress.certManager }}
+    kubernetes.io/tls-acme: "true"
```

Comment:
In other ingress files we have:
```
    kubernetes.io/ingress.class: nginx
    {{ if .Values.apiIngress.clusterIssuer }}
    cert-manager.io/cluster-issuer: {{ .Values.apiIngress.clusterIssuer }}
    {{ end }}
```
Is this equivilent?

Reply:
The tls-acme annotation seems to be deprecated, so removing it and moving to the pattern being used in apiIngress